diff --git a/node_modules/react-native-network-logger/src/XHRInterceptor.ts b/node_modules/react-native-network-logger/src/XHRInterceptor.ts
index 69e7623..4e9fcd3 100644
--- a/node_modules/react-native-network-logger/src/XHRInterceptor.ts
+++ b/node_modules/react-native-network-logger/src/XHRInterceptor.ts
@@ -11,15 +11,21 @@ type XHRInterceptorModule = {
 
 let XHRInterceptor: XHRInterceptorModule;
 try {
-  // new location for React Native 0.79+
-  const module = require('react-native/src/private/inspector/XHRInterceptor');
+  // new location for React Native 0.80+
+  const module = require('react-native/src/private/devsupport/devmenu/elementinspector/XHRInterceptor');
   XHRInterceptor = module.default ?? module;
 } catch {
   try {
-    const module = require('react-native/Libraries/Network/XHRInterceptor');
+    // new location for React Native 0.79+
+    const module = require('react-native/src/private/inspector/XHRInterceptor');
     XHRInterceptor = module.default ?? module;
   } catch {
-    throw new Error('XHRInterceptor could not be found in either location');
+    try {
+      const module = require('react-native/Libraries/Network/XHRInterceptor');
+      XHRInterceptor = module.default ?? module;
+    } catch {
+      throw new Error('XHRInterceptor could not be found in either location');
+    }
   }
 }
 
