diff --git a/node_modules/react-native-network-logger/src/Logger.ts b/node_modules/react-native-network-logger/src/Logger.ts
index ffe8dd9..6dcc8ca 100644
--- a/node_modules/react-native-network-logger/src/Logger.ts
+++ b/node_modules/react-native-network-logger/src/Logger.ts
@@ -1,4 +1,4 @@
-import XHRInterceptor from 'react-native/Libraries/Network/XHRInterceptor';
+import XHRInterceptor from 'react-native/src/private/inspector/XHRInterceptor';
 import NetworkRequestInfo from './NetworkRequestInfo';
 import { Headers, RequestMethod, StartNetworkLoggingOptions } from './types';
 import extractHost from './utils/extractHost';
